module-apps/apps-common/widgets/TimeFixedWidget.hpp

```diff
@@ -21,21 +48,26 @@ namespace gui
                         const uint32_t &h,
                         const bool minus = false);
 
-        void setFirst(uint32_t first);
-        void setSecond(uint32_t second);
-        void setMinus(bool minus);
-        void setFont(const UTF8 &fontName);
+        void setMinutesBox(uint32_t first) const;
+        void setSecondsBox(uint32_t second) const;
+        void setFontAndDimensions(const UTF8 &fontName) const;
+
+        const LeftBox &getLeftBox() const;
```

Comment:
I believe these methods were only added for testing purposes? Imho it's a bad practice to do such things.

Reply:
I would disagree that adding functionality for test purpouse is bad practice. I could return state by already existing function but I'm wondering if its better. Imo less intuitive but whatever I can do not use these getter functions

Reply:
Another way would be mock injection but it would complicate code and test quite much

Reply:
I'm not sure if leaking internals via public API is considered a good practice. As a result of this, changes to the internals will affect existing tests and in the end make them more unstable.

Reply:
Note that LeftBox/RightBox contains manully allocated memory so copy the would by risky(would require copty-ctor and so on).
What would would be better alternative in this case?